docs/en/siem/siem-ui.asciidoc

```diff
@@ -204,24 +205,35 @@ that aren't present?  Let us know. We welcome your input.
 Depending on your {kib} setup, to display and interact with data on the map you
 might need to:
 
+* <<kibana-index-pattern>>
 * <<geoip-data, Add geographical IP data to events>>
 * <<private-network>>
 
 NOTE: To see source and destination connections lines on the map, you must
 configure `source.geo` and `destination.geo` ECS fields for your indices.
 
+[float]
+[[kibana-index-pattern]]
+==== Create {kib} index patterns
+
+To display data on the map, you must define {kib} index patterns 
+(Management -> Index Patterns) that match all the {siem-soln} {es} indices you 
```

Comment:
Only note here is whether _'match'_ could be understood to mean `pattern matching` vs `exact string match`. Not sure if clarification is needed, but I think this may be the last area of confusion (🤞)  until we [support pattern matching](https://github.com/elastic/kibana/issues/52565) when choosing which layers to create

Reply:
Yes, thanks. Updating the text.